Resources/Prototypes/_FarHorizons/Entities/Objects/Devices/pda.yml

```
@@ -115,3 +115,130 @@
       whitelist:
         tags:
         - Write
+
+#SYNDI OPERATIVE PDA - needed for nukie pdas so they dont have sensors that show crew where they are
+
+- type: entity
+  abstract: true
+  parent: [ BaseItem, StorePresetUplink ] 
+  id: BaseSyndiOperativePDA
```

Since we will be having NT Nuke Ops in the future it would be good to have the name on this neutrally NukeOpsPDA.
```suggestion
  id: BaseNukeOpsPDA
```